renderdoc/driver/ihv/intel/intel_counters.cpp

```diff
@@ -0,0 +1,423 @@
+/*****************************************************************************\
+
+  Copyright (C) 2018, Intel Corporation
```

Comment:
All code contributed to renderdoc should follow the [copyright / contributor license agreement in CONTRIBUTING.md](https://github.com/baldurk/renderdoc/blob/v1.x/CONTRIBUTING.md#copyright--contributor-license-agreement) and assign copyright to the project owner myself. The license at the top of these files should be identical to any other renderdoc source aside from the year.